ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml

```diff
@@ -1297,29 +1297,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
-- as: gcp-ipi-cco-manual-workload-identity-p2-f14
-  cron: 45 19 8,23 * *
+- as: gcp-ipi-disconnected-private-cco-manual-sts-p3-f28
+  cron: 14 6 19 * *
   remote_api: true
   steps:
     cluster_profile: gcp-qe
     env:
-      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @network-openshiftsdn and not @fips'
+      E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @disconnected and @network-openshiftsdn
+        and not @fips'
       TAG_VERSION: '@4.10'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
-- as: gcp-ipi-cco-manual-workload-identity-p3-f28-destructive
```

Comment:
Can we keep the destructive testing and update it to test cucushift-installer-rehearse-gcp-ipi-disconnected-private-cco-manual-workload-identity ?

Reply:
Once that, I guess we can remove gcp-ipi-disconnected-private-p3-f28-destructive